lib/src/layout/resizable_pane.dart

```diff
@@ -48,14 +50,46 @@ class ResizablePane extends StatefulWidget {
         assert(
           (startSize >= minSize) && (startSize <= maxSize),
           'startSize must not be less than minSize or more than maxWidth',
-        );
+        ) {
+    useScrollBar = true;
+  }
+
+  /// Create a [ResizablePane] without an internal [MacosScrollbar]
+  /// {@macro resizablePane}
```

Comment:
```suggestion
  /// Creates a [ResizablePane] without an internal [MacosScrollbar].
  /// {@macro resizablePane}.
```
In between these 2 lines, please add some documentation that describes why this constructor may be useful. It may not be intuitive for some, so docs are preferred.